awx/settings/defaults.py

```diff
@@ -1079,7 +1079,7 @@
 
 
 # setting for Policy as Code feature
-FEATURE_POLICY_AS_CODE_ENABLED = False
+FEATURE_POLICY_AS_CODE_ENABLED = True
```

Comment:
Why have this setting for PoC, if not indirect node counting?

I mean, why does this not have:

```python
FEATURE_INDIRECT_NODE_COUNTING_ENABLED = False
```

The function this line is playing is still very unclear to me. Would it work the same if this line wasn't here?